ChronoKeeper/StoryChunk.h

```diff
@@ -45,6 +46,17 @@ class StoryChunk
     bool empty() const
     { return (logEvents.empty() ? true : false); }
 
+    size_t getNumEvents() const
+    { return logEvents.size(); }
+
+    size_t getTotalPayloadSize() const
```

Comment:
> I suggest getTotalPayloadSize() is implemented differently in the subsequent version. Instead of iterating over the StoryChunk every time the method is called, there should be total_size value that's a member of the class and it's adjusted properly when records are added to and removed from StoryChunk. we tackle this as part of next release integration

Rolled back in the latest commit.